pkg/engine/virtualmachines.go

```diff
@@ -563,3 +565,9 @@ func getVaultSecretGroup(linuxProfile *api.LinuxProfile) []compute.VaultSecretGr
 	}
 	return vaultSecretGroups
 }
+
+func addCustomTagsToVM(cs *api.ContainerService, vm *compute.VirtualMachine) {
```

Comment:
let's unit test this (and the VMSS equivalent)

Reply:
I'll create the tests :)